.kiro/specs/auterity-ai-platform-unified/templates/enhanced-workflow-canvas-template.tsx

```
@@ -0,0 +1,746 @@
+              </div>
+            )}
+          </Suspense>
+        );
+
+      case NodeType.CONTENT:
+        return (
+          <div className="h-64 overflow-auto">
+            {previewResult.content && (
+              <div 
+                className="prose prose-sm max-w-none"
+                dangerouslySetInnerHTML={{ __html: previewResult.content.rendered }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)